rotkehlchen/errors/misc.py

```diff
@@ -80,3 +80,11 @@ class DBSchemaError(Exception):
 
 class GreenletKilledError(Exception):
     """Raised when a greenlet is killed"""
+
+
+class AccountingError(Exception):
+    """Error processing the """
+
+    def __init__(self, message: str = '', report_id: int | None = None):
```

Comment:
Why message is an optional argument? It should not be.

As for report_id, please explain in the docstring it's not visible in the place where error is raises and is later set when caught.